src/developers/notary_server.md

```diff
@@ -67,7 +67,8 @@ We host multiple versions of the notary server:
 
 | Version       | Notary URL                            | Info/Status                                                                                                    | GitHub                                                                                     | Note                                          |
 |---------------|---------------------------------------|----------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------|-----------------------------------------------|
-| latest        | https://notary.pse.dev                | [info](https://notary.pse.dev/info)/[health](https://notary.pse.dev/healthcheck)                               | [main](https://github.com/tlsnotary/tlsn/tree/main/notary-server)                          | Currently the same as v0.1.0-alpha.4           |
+| v0.1.0-alpha.5 | https://notary.pse.dev/v0.1.0-alpha.5 | [info](https://notary.pse.dev/v0.1.0-alpha.5/info)/[health](https://notary.pse.dev/v0.1.0-alpha.5/healthcheck) | [v0.1.0-alpha.5](https://github.com/tlsnotary/tlsn/tree/v0.1.0-alpha.4/notary-server)       | [Release notes](https://github.com/tlsnotary/tlsn/releases/tag/v0.1.0-alpha.4) |
```

Comment:
we'll be pointing `main` to v5 after release right? if so, `GitHub` value for v5 and deprecated need to change

Reply:
where for v5 it should be `main` and for deprecated it should be `v0.1.0-alpha.4`